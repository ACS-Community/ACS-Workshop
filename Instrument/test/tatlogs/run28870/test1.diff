1,5c1,4
< 1 - 
< 1 - ----------------------------------------------------------------------
< 1 - .
< 1 - CAMERA ON
< 1 - OK
---
> 1 -                         ^
> 1 -     def testCamera(self):
> 1 -   File "/home/teami/ACS-Workshop/Instrument/bin/test1", line 22
> 1 - IndentationError: unindent does not match any outer indentation level

