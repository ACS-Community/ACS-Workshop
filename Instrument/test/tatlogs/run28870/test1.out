1 -   File "/home/teami/ACS-Workshop/Instrument/bin/test1", line 22
1 -     def testCamera(self):
1 -                         ^
1 - IndentationError: unindent does not match any outer indentation level
